src/SmartComponents/SystemAdvisor/SystemAdvisor.js

```diff
@@ -517,5 +517,19 @@ const SystemAdvisor = ({ ...props }) => {
   return <BaseSystemAdvisor {...props} entity={entity} />;
 };
 
+const SystemAdvisorWrapped = ({ ...props }) => {
```

Comment:
For the consistency sake - lets make a new file for the SystemsAdvisorWrapped as we did for other components.

Reply:
Done